src/commands/troll/hitlist.ts

```diff
@@ -0,0 +1,42 @@
+import { KillIntervals } from "../../custom/storage";
+import {
+    CommandoMessage,
+    Command,
+    CommandoClient,
+    ArgumentInfo,
+} from "discord.js-commando";
+
+module.exports = class HitListCommand extends (Command) {
+    constructor(client: CommandoClient) {
+        super(client, {
+            name: "hitlist",
+            group: "troll",
+            memberName: "hitlist",
+            aliases: ["hl", "killList"],
+            description: "Displays killed people in the server in a hitlist.",
+        });
+    }
+
+    async run(message: CommandoMessage, args: any) {
+        return message.say("People currently killed in the server: " + killedPeople());
+    }
+};
+
+
+/**
+ * Returns values of killed users of a server in an array.
+ * @returns {string[]} array of killed users in a server.
+ */
+
+function killedPeople() {
+    const intervals = KillIntervals.getInstance();
```

Comment:
The function signature is still missing a return type annotation.
`function fnName():  returnType {
   ...
}`